doc/sphinx/Clusters_from_Scratch/ap-configuration.rst

```diff
@@ -0,0 +1,372 @@
+Configuration Recap
+-------------------
+
+Final Cluster Configuration
+###########################
+
+::
+
+    [root@pcmk-1 ~]# pcs resource
+     Master/Slave Set: WebDataClone [WebData]
+         Masters: [ pcmk-1 pcmk-2 ]
+     Clone Set: dlm-clone [dlm]
+         Started: [ pcmk-1 pcmk-2 ]
+     ClusterIP	(ocf::heartbeat:IPaddr2):	Started pcmk-1
+     Clone Set: WebFS-clone [WebFS]
+         Started: [ pcmk-1 pcmk-2 ]
+     WebSite	(ocf::heartbeat:apache):	Started pcmk-1
+
+::
+
+    [root@pcmk-1 ~]# pcs resource op defaults
+    timeout: 240s
+
+::
+
+    [root@pcmk-1 ~]# pcs stonith
+     impi-fencing	(stonith:fence_ipmilan): Started pcmk-1
+
+::
+
+    [root@pcmk-1 ~]# pcs constraint
+    Location Constraints:
+    Ordering Constraints:
+      start ClusterIP then start WebSite (kind:Mandatory)
+      promote WebDataClone then start WebFS-clone (kind:Mandatory)
+      start WebFS-clone then start WebSite (kind:Mandatory)
+      start dlm-clone then start WebFS-clone (kind:Mandatory)
+    Colocation Constraints:
+      WebSite with ClusterIP (score:INFINITY)
+      WebFS-clone with WebDataClone (score:INFINITY) (with-rsc-role:Master)
+      WebSite with WebFS-clone (score:INFINITY)
+      WebFS-clone with dlm-clone (score:INFINITY)
+    Ticket Constraints:
+
+::
+
+    [root@pcmk-1 ~]# pcs status
+    Cluster name: mycluster
+    Stack: corosync
+    Current DC: pcmk-1 (version 1.1.18-11.el7_5.3-2b07d5c5a9) - partition with quorum
+    Last updated: Tue Sep 11 10:41:53 2018
+    Last change: Tue Sep 11 10:40:16 2018 by root via cibadmin on pcmk-1
+
+    2 nodes configured
+    11 resources configured
+
+    Online: [ pcmk-1 pcmk-2 ]
+
+    Full list of resources:
+
+     ipmi-fencing   (stonith:fence_ipmilan):        Started pcmk-1
+     Master/Slave Set: WebDataClone [WebData]
+         Masters: [ pcmk-1 pcmk-2 ]
+     Clone Set: dlm-clone [dlm]
+         Started: [ pcmk-1 pcmk-2 ]
+     ClusterIP	(ocf::heartbeat:IPaddr2):	Started pcmk-1
+     Clone Set: WebFS-clone [WebFS]
+         Started: [ pcmk-1 pcmk-2 ]
+     WebSite	(ocf::heartbeat:apache):	Started pcmk-1
+
+    Daemon Status:
+      corosync: active/disabled
+      pacemaker: active/disabled
+      pcsd: active/enabled
+
+::
+
+    [root@pcmk-1 ~]# pcs cluster cib --config
+
+.. code:: xml
```

Comment:
code-block (unless that's another difference between versions?)

Reply:
No, just a thinko from running through all these files in a day.